projects/ngx-d3timeline/src/lib/content/selectors/activity.selectors.ts

```diff
@@ -46,8 +53,39 @@ const selectDraggedToSeries = createSelector(
   valueToSeries
 );
 
+const selectDeltaTime = createSelector(
+  selectTimeOrientation,
+  selectDragEvent,
+  getDeltaTime
+);
+
+const selectCurrentlyDraggedUpdatedStart = createSelector(
+  selectCurrentlyDraggedActivity,
+  activity => activity && activity.updatedStart
+);
+
+const selectCurrentlyDraggedUpdatedFinish = createSelector(
+  selectCurrentlyDraggedActivity,
+  activity => activity && activity.updatedFinish
+);
+
+const selectShiftedByDeltaTime = (selectTime: MemoizedSelector<Date>) =>
+  createSelector(
+    selectTime,
+    selectTimeScale,
+    selectDeltaTime,
+    shiftTimeByRangeValue
+  );
+
 export const selectCurrentlyDraggedActivityWithDraggedToSeries = createSelector(
   selectCurrentlyDraggedActivity,
   selectDraggedToSeries,
   getCurrentlyDraggedActivityWithDraggedToSeries
 );
+
+export const selectDraggedToActivity = createSelector(
+  selectCurrentlyDraggedActivityWithDraggedToSeries,
```

Comment:
Not sure of the need for this selector, its getting the dragged activity with the updated series, then the 2 selectors are getting updated times, then its put together. So one drag update field (series) is nested and the other two (times) aren't.

Why not have all the updates as selectors here then combine at the end?
```
selectCurrentlyDraggedActivity,
selectDraggedToSeries,
selectShiftedByDeltaTime(selectCurrentlyDraggedUpdatedStart),
selectShiftedByDeltaTime(selectCurrentlyDraggedUpdatedFinish),
updateActivityForDrag
```

Reply:
done